packages/http/package.json

```
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@stoplight/prism-core": "0.0.0",
-    "ajv": "6.6.x",
+    "ajv": "6.10.0",
```

why not `6.10.x`?

---

`yarn` sets this automatically, but I'm ok with changing it
```suggestion
    "ajv": "6.10.x",
```